src/ripple/nodestore/impl/DatabaseShardImp.h

```diff
@@ -137,18 +139,23 @@ class DatabaseShardImp : public DatabaseShard
     getWriteLoad() const override;
 
     void
-    store(NodeObjectType type, Blob&& data,
-        uint256 const& hash, std::uint32_t seq) override;
+    store(
+        NodeObjectType type,
+        Blob&& data,
+        uint256 const& hash,
+        std::uint32_t seq) override;
 
     std::shared_ptr<NodeObject>
     fetch(uint256 const& hash, std::uint32_t seq) override;
 
     bool
-    asyncFetch(uint256 const& hash, std::uint32_t seq,
+    asyncFetch(
+        uint256 const& hash,
+        std::uint32_t seq,
         std::shared_ptr<NodeObject>& object) override;
 
     bool
-    copyLedger(std::shared_ptr<Ledger const> const& ledger) override;
+    copyLedger(std::shared_ptr<Ledger const> const& srcLedger) override;
```

Comment:
I named it `copyLedger` because it copies a ledger from a source database to this one. The function will be deprecated and removed when we transition to having the shard store substitute the node store. Considering the functionality, I am not certain `addLedger` or `incorporateLedger` are ideal. Because it is a database, `storeLedger` may be a good option. What do you think?

Reply:
Yeah, `storeLedger` seems like an excellent choice.  Good suggestion.